Library/Homebrew/utils/formatter.rb

```diff
@@ -90,4 +90,20 @@ def columns(*objects, gap_size: 2)
 
     output
   end
+
+  def pluralize(count, singular, plural = nil)
```

Comment:
I'd have probably flipped `count` and `singular` but don't have strong feelings.